src/mixins/people-roles-mixin.ts

```diff
@@ -25,12 +25,12 @@ export default class PeopleRolesMixin extends Vue {
   @Getter(useStore) getUserEmail!: string
   @Getter(useStore) getUserFirstName!: string
   @Getter(useStore) getUserLastName!: string
+  @Getter(useStore) isEntityPartnership!: boolean
+  @Getter(useStore) isEntitySoleProp!: boolean
```

Comment:
These getters are not used in this mixin, but this mixin is used by a couple of components that use the getters.